beluga/routes.py

```diff
@@ -4,8 +4,12 @@
 from sanic.exceptions import abort
 from sanic.response import json
 
+from beluga.models import session_scope, Event
 from beluga.auth import authorized
 
+import json as pyjson
+from datetime import timezone as tz
+import math
```

Comment:
[Nitpick] Idiomatic imports: alphabetic and using 3 blocks: standard libraries, 3rd party, local modules.
These three new ones are standard libraries, so put em at the top ☝️.

Reply:
will do